src/clr-angular/forms/datepicker/model/calendar-view.model.ts

```diff
@@ -80,12 +82,32 @@ export class CalendarViewModel {
     this.initializeFocusableDay();
   }
 
+  private isDateDisabled(date: DayModel) {
+    let disabled = false;
```

Comment:
I think that this could be written in a more readable way. Right now is really hard to wrap around it.

```typescript
private isDateDisabled(date: DayModel) {
  const { bottom, top } = this.disabledDates;
  const from = new Date(bottom.year, bottom.month - 1, bottom.date);
  const to = new Date(top.year, top.month - 1, top.date);
  const today = new Date(date.year, date.month - 1, date.date);
  return !(today >= from && today <= to);
}
```

You could go with this but I think some of the readability will be lost
```typescript
private isDateDisabled(date: DayModel) {
  const { bottom, top } = this.disabledDates;
  const today = new Date(date.year, date.month - 1, date.date);
  return !(
      today >= (new Date(bottom.year, bottom.month - 1, bottom.date)) 
      && 
      today <= (new Date(top.year, top.month - 1, top.date))
  );
}
```